### PR TITLE
Redirect to anchor when updating ClinVar Casedata and improved submission howto

### LINKED ISSUE #2443: cvs file for ClinVar submission (to Chiara)
instructions on how to handle the files downloaded from SCout for CLinVar submission 

[ClinVar instruction for .cvs files from Scout.pptx](https://github.com/Clinical-Genomics/scout/files/6148781/ClinVar.instruction.for.cvs.files.from.Scout.pptx)

For users have a Swedish PC with Windows 10

### LINKED ISSUE #2468: ClinVar - jumping page - Q of life
image

Hi.
Two small problems

1. when I save the name change the page jump to the top. I need to scroll down every name change (5 time in this case) . It is possible to make the page not to move? In case of large number of variants it can become quite a bit of extra work  :)

2. If I write all 5 namne changes at once, Then I save one.  the other 4 go back to the initial name- This is Ok if it is not solved

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR:
- Makes the vertical scrollable div of the ClinVar submissions page a bit higher.
- Introduces redirect to the specific submission casedata table when a user modifies the name of submission cases (fix #2468)
- Improves submission instructions for users who edit exported files with a non-English Excel software (fix #2443):

image

**How to test**:
1.  Create more than 1 ClinVar submission (remember to pin variants to see the option).
2. Edit the names of the individuals in casedata of one of the submissions located at the bottom of the page and make sure that after saving the new names the page redirects to the same panel.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html [hunk 1]
```diff
@@ -177,60 +179,60 @@ <h4>Variant data:</h4>
...
-                <td><a href="{{ url_for('cases.case', institute_id=institute._id, case_name=subm_obj.cases[case.case_id]) }}" target="_blank">{{subm_obj.cases[case.case_id]}}</a></td>
-                <td>{{case.clin_features}}</td>
-                <td>{{var_key_name[case.linking_id]}}</td>
-                <td>{{case.allele_origin}}</td>
-                <form id="delete_casedata_{{case._id}}" action="{{ url_for('overview.clinvar_delete_object', submission=subm_obj._id, object_type='case_data') }}" method="POST">
-                  <td><button type="submit" name="delete_object" value="{{case._id}}" class="btn btn-danger btn-xs"><span class="fa fa-trash-o fa-lg" aria-hidden="true"></span></button></td>
-                </form>
-              </tr>
+      {% if subm_obj.case_data and subm_obj.case_data | count_cursor > 0 %}
+        <div id="cdata_{{subm_obj._id}}"> <!--case data div -->
```
Comment (northwestwitch) [inline scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html:183]: This document seems a lot different but the only things I did is adding an ID to this div and moving the div inside the 
`{% if subm_obj.case_data and subm_obj.case_data | count_cursor > 0 %}` condition

File: scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html [hunk 2]
```diff
@@ -22,8 +22,8 @@
 <div class="container-float">
   <h3>ClinVar submissions page<h5></h3><a data-target="#howto" href="#" data-toggle="modal">howto</a></h5><br>
   <!-- Modal -->
-  <div class="modal fade w-100" role="dialog" id="howto" aria-hidden="true" tabindex="-1">
-    <div class="modal-dialog" role="document">
+  <div class="modal fade bd-example-modal-lg" role="dialog" id="howto" aria-hidden="true" tabindex="-1">
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html:25]: `bd-example-modal-lg`?

File: scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html [hunk 3]
```diff
@@ -45,10 +45,12 @@ <h5>Download the files</h5>
           <p>
             You'll need to register the <strong>Submission Name</strong> obtained from the ClinVar portal in order to download variants and casedata submission files.
           </p><br>
-          <h5>Upload the files to Clinvar</h5>
+          <h5>Upload the files to ClinVar</h5>
           <p>
             Upload the files obtained above (either the variants .csv file or the variants .csv + the casedata .csv files) in the "<strong>Submission Files</strong>" field in the third panel of the submission.
-          The field named <strong>"Assertion Criteria File" should be left blank</strong>, since the default information is already filled in the "Variants sheet" <strong>(ACMG Guidelines, 2015)</strong>.
+          The field named <strong>"Assertion Criteria File" should be left blank</strong>, since the default information is already filled in the "Variants sheet" <strong>(ACMG Guidelines, 2015)</strong>.<br>
+          <text class="text-danger"> Note that if you edit these files in Excel you need to save them in the correct format: <strong>comma-separated columns</strong>. Default settings should probably work fine if your software's locale is US English.
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html:52]: 👍

**Review by dnil — APPROVED** [review]

File: scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html:25]: That would be the data target of the example I think, will fix!

Comment (northwestwitch) [reply scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html]: Fixed, and made the modal extra large:

image

**Pull request merged**